coreos-4/policies/SC-Systems_and_Communications_Protection/component.yaml

```diff
@@ -8,4 +8,1548 @@
   covered_by: []
   implementation_status: not applicable
   narrative:
-    - text: ''
+    - key: a
+      text: |
+        This control reflects organizational procedures/policies, and is not
+        applicable to the configuration of CoreOS.
+    - key: b
+      text: |
+        This control reflects organizational procedures/policies, and is not
+        applicable to the configuration of CoreOS.
+
+- control_key: SC-2
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: planned
+  narrative:
+    - text: |
+        The customer is responsible for separating user functionality
+        including user interface services) from information system management
+        functionality. A successful control response will indicate how user
+        and management functionality access is separated.
+
+        A complete control response is planned. Engineering progress can be
+        tracked via:
+
+        https://issues.redhat.com/browse/CMP-429
+
+- control_key: SC-2 (1)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: planned
+  narrative:
+    - text: |
+        A complete control response is planned. Engineering progress can be
+        tracked via:
+
+        https://issues.redhat.com/browse/CMP-466
+
+- control_key: SC-3
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: planned
+  narrative:
+    - text: |
+        The customer is responsible for deployment utilizing namespaces,
+        host roles and other features as well as policies and procedures
+        to implement and ensure continued isolation of security functions
+        from non-security functions.
+        A complete control response is planned. Engineering progress can be
```

Comment:
Few occurrences like this. Should be another new line added.

Reply:
These are fixed... not sure why they are not showing the updated fixes.